.github/workflows/submodule_check.yaml

```diff
@@ -0,0 +1,24 @@
+name: Check rive-wasm sub modules
+
+on:
+  # lets check it when we push to master (we've just done something in android, its a good time to see about this)
+  push:
+    branches:
+      - master
+  
+  schedule:
+    # 8am UTC every day
+    - cron:  '0 8 * * *'
+
+
+jobs:
+  build:
+    runs-on: macos-latest
```

Comment:
Just noticing that this is running on macos, and I think we've got much less free compute time for these machines (i.e. they're much more expensive). Any reason to run them on mac rather than linux?

Reply:
I think we have a number of free compute minutes each month, and after a while they run out.
In fact, [macOS has a higher multiplier minutes](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions#about-billing-for-github-actions) which might make them run out faster.

Reply:
Oh, you're right! Nvmd then! Free 🍻